### PR TITLE
feature/CC-2474: Adding endpoint for hosts with reduced payload.

### PR DESCRIPTION
**Pull request by cchristiezen**
Adding a new v2 endpoint for hosts.  The endpoint returns hosts with only the properties necessary for the hosts page.  Reducing the payload (for example, monitoring profile is not included) to only those properties should help make the hosts page faster for sites with large amounts of hosts.

### CODE REVIEW

File: domain/host/host.go [hunk 1]
```diff
@@ -72,6 +72,27 @@ type Host struct {
...
+	Memory        uint64
+	RAMLimit      string
+	KernelVersion string
+	KernelRelease string
+	ServiceD      ReadServiceD
+	CreatedAt     time.Time
+	UpdatedAt     time.Time
+}
+
+type ReadServiceD struct {
```
Comment (smousa) [inline domain/host/host.go]: don't capitalize the d

File: web/api_hosts.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package web
```
Comment (smousa) [inline web/api_hosts.go]: boilerplate

File: web/api_hosts_test.go [hunk 3]
```diff
@@ -0,0 +1,157 @@
+// Copyright 2014 The Serviced Authors.
```
Comment (smousa) [inline web/api_hosts_test.go]: Copywrite year

**Comment by cchristiezen** [comment]
retest this

**Pull request merged**